### PR TITLE
V3

### PR DESCRIPTION
**Pull request by zxan1285**
## ✏️ Changes
Removed cipher support (see https://auth0team.atlassian.net/browse/DXEX-234)
Added ability to exclude `clients`, `databases` and `connections`
Added mappings support
Added mechanism to automatically re-deploy last successful configuration in case of failed deployment
  
## 📷 Screenshots
Screen Shot 2019-08-20 at 12 33 32

## 🔗 References
  Jira: https://auth0team.atlassian.net/browse/DXEX-234
  
## 🎯 Testing
✅ This change has been tested in a Webtask
✅ This change has unit test coverage
🚫 This change has integration test coverage
🚫 This change has been tested for performance
  
## 🚀 Deployment
✅ This can be deployed any time

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -20,22 +20,22 @@ jobs:
             # fallback to using the latest cache if no exact match is found
             - npm-dependencies-
 
-      - run:
-          name: Install FOSSA
-          command: |
-            curl -H 'Cache-Control: no-cache' https://raw.githubusercontent.com/fossas/fossa-cli/master/install.sh | bash
+      #- run:
```
Comment (lzychowski) [inline .circleci/config.yml:23]: any reason to keep the commented out stuff?

File: client/actions/excludes.js [hunk 2]
```diff
@@ -4,24 +4,24 @@ import * as constants from '../constants';
 /*
  * Load the rules.
  */
-export function fetchAllRules() {
+export function fetchExcludes() {
```
Comment (lzychowski) [inline client/actions/excludes.js:7]: why did we rename the rules file when adding multiple excludes?

File: .circleci/config.yml (hunk 1 above)
Comment (zxan1285) [reply .circleci/config.yml:23]: I'm not sure, but i guess at some point we may want to fix that fossa stuff.

File: client/actions/excludes.js (hunk 2 above)
Comment (zxan1285) [reply client/actions/excludes.js:7]: Originally, there weren't any other excludes beside rules. Then resourceServers was added and we just copied excludeRules component. Now we have rules, resourceServers, clients, connections and databases and all are using the same code. So i removed `resourceServers` excludes component and changed `rules` to general `excludes`.

**Pull request merged**